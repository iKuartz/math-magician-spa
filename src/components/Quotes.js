const Quotes = () => (
  <div className="quotesDiv">
    <p className="quoteParagraph">
      Logic and mathematics are nothing but specialized linguistic structures. -- Jean Piaget
    </p>
  </div>
);

export default Quotes;
