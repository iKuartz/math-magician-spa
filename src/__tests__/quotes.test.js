import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

it('Checking if Quotes page renders correctly', () => {
  const tree = renderer
    .create(
        <Quotes />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});