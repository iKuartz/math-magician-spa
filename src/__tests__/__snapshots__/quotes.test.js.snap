// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking if Quotes page renders correctly 1`] = `
<div
  className="quotesDiv"
>
  <p
    className="quoteParagraph"
  >
    Logic and mathematics are nothing but specialized linguistic structures. -- Jean Piaget
  </p>
</div>
`;
